udfs/community/get_value.sql

```diff
@@ -0,0 +1,5 @@
+-- Given a key and a map, returns the SCALAR type value.
```

Comment:
I find this comment confusing along with the `arr` variable name since here we say the functions takes a map but it actually takes a list of maps.

Reply:
LMK if this is more clear

Reply:
LGTM